XenAdmin/Controls/PvsCacheStorageRow.cs

```diff
@@ -95,7 +97,10 @@ private void PopulateCacheSrCombobox()
                     opaque_ref = Helper.NullOpaqueRef
                 };
             }
-            var memorySrItem = new ToStringWrapper<SR>(memorySr, Messages.PVS_CACHE_MEMORY_ONLY);
+
+            var enabled = Host.dom0_memory_extra >= numericUpDownCacheSize.Minimum;

```

Comment:
Also, we could use the MIN_CACHE_SIZE_GB constant instead of numericUpDownCacheSize.Minimum